appsdk/configurable_test.go

```diff
@@ -174,7 +174,10 @@ func TestConfigurableMQTTSend(t *testing.T) {
 	}
 
 	params := make(map[string]string)
-	addr := models.Addressable{}
+	addr := models.Addressable{
+		Address: "http://url",
```

Comment:
This should be `tcp` for MQTT. Again this code is for MQTT Export, so this change is out of scope for this PR.